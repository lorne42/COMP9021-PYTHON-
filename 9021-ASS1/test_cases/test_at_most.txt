I ran into Sirs Eve and Frank.
Sir Eve said: "At most one of us is a Knight."
Sir Frank said: "I am a Knave."
Who is a Knight and who is a Knave?
