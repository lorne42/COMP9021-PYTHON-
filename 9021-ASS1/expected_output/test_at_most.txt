The Sirs are: Eve Frank
There is no solution.